### PR TITLE
Khn/master

### PR DESCRIPTION
**Pull request by knguyen0710**
This submission is for the breakfast tier.

For the ".highest_score_from" I was able to complete all requirements for the method, though in a round-a-bout way. I commented out a previous attempt on that method that was much more concise but that was not able to successfully complete to account for shorter words.

For my spec file, I was able to write functionality tests for a range of methods, but again, given more time, I'd add a few more tests for ".highest_score_from".

### CODE REVIEW

File: spec/lib/scrabble_spec.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
+require './lib/scrabble/scrabble.rb'
+
+describe Scrabble::Scrabble do
```
Comment (mhotan) [inline spec/lib/scrabble_spec.rb:3]: I like the structure of the test.  Clear and easy to read what you are testing.  As another developer I love that.  What can be improved is testing the highest_score_from method.  I would like to see tests for sets of words that meet certain boundary cases (not equal in point value, equal in point but not in length, equal in point and length).

Comment (knguyen0710) [inline spec/lib/scrabble_spec.rb:3]: @mhotan I'm not sure what you mean. So right now, the highest_score_from method handles all the cases you mentioned. For not equal in point value, it only returns the highest point scoring word. For equal in point, it checks for the length of the word and returns either the shortest or the 7 letter word. And for equal in point and length, it returns the first word as a side effect of the minmax_by method.

Comment (mhotan) [inline spec/lib/scrabble_spec.rb:3]: Hmm maybe I am missing something.  Lets take a couple minutes in class and walk me through your test cases.   I feel like we are reaching the limits of what can be conveyed through github messages.
